### PR TITLE
[v6r22] fixes for logging messages for Matcher

### PR DESCRIPTION
**Pull request by fstagni**
BEGINRELEASENOTES

*WMS
FIX: fixes for logging messages for Matcher
FIX: fixes for logging messages for Optimizers

ENDRELEASENOTES

### CODE REVIEW

File: WorkloadManagementSystem/Client/Matcher.py [hunk 1]
```diff
@@ -334,7 +335,8 @@ def _checkCredentials(self, resourceDict, credDict):
             raise RuntimeError(result['Message'])
           if credDict['group'] not in result['Value']:
             # DN is not in the same group! bad boy.
-            self.log.notice("You cannot request jobs from DN %s. It does not belong to your group!" % ownerDN)
+            self.log.notice("You cannot request jobs from this DN, as it does not belong to your group!",
```
Comment (chaen) [inline WorkloadManagementSystem/Client/Matcher.py]: Should this message really be `notice` ?

File: WorkloadManagementSystem/Executor/JobPath.py [hunk 2]
```diff
@@ -53,7 +53,7 @@ def __executeVOPlugin(self, voPlugin, jobState):
       modInstance = self.__voPlugins[voPlugin](argsDict)
       result = modInstance.execute()
     except Exception as excp:
-      self.jobLog.exception("Excp while executing %s" % voPlugin)
+      self.jobLog.exception("Excp while executing", voPlugin)
```
Comment (chaen) [inline WorkloadManagementSystem/Executor/JobPath.py]: you may want to print the exception too ?

File: WorkloadManagementSystem/Executor/JobPath.py [hunk 3]
```diff
@@ -103,7 +103,7 @@ def optimizeJob(self, jid, jobState):
       if opN not in uPath:
         uPath.append(opN)
     opPath = uPath
-    self.jobLog.info('Constructed path is: %s' % "->".join(opPath))
+    self.jobLog.info('Constructed path is' ': %s' % "->".join(opPath))
```
Comment (chaen) [inline WorkloadManagementSystem/Executor/JobPath.py]: there is a comma missing to separate the parameters

File: tests/Jenkins/dirac_ci.sh [hunk 4]
```diff
@@ -131,7 +131,7 @@ function installSite(){
 
   echo '==> Started installing'
 
-  installOptions="-t server $DEBUG "
+  installOptions="$DEBUG "
```
Comment (chaen) [inline tests/Jenkins/dirac_ci.sh]: why removing the server option from there ?

File: WorkloadManagementSystem/Client/Matcher.py (hunk 1 above)
Comment (fstagni) [reply WorkloadManagementSystem/Client/Matcher.py]: `warn`?

File: WorkloadManagementSystem/Executor/JobPath.py (hunk 2 above)
Comment (fstagni) [reply WorkloadManagementSystem/Executor/JobPath.py]: the stacktrace of the exception will always be printed.

File: tests/Jenkins/dirac_ci.sh (hunk 4 above)
Comment (fstagni) [reply tests/Jenkins/dirac_ci.sh]: `fullServer` option is added when DIRACOS is not used.

File: WorkloadManagementSystem/Executor/JobPath.py (hunk 2 above)
Comment (chaen) [reply WorkloadManagementSystem/Executor/JobPath.py]: ah true !

File: WorkloadManagementSystem/Client/Matcher.py (hunk 1 above)
Comment (chaen) [reply WorkloadManagementSystem/Client/Matcher.py]: I don't know, I did not check the logic, but being forbidden to do something and being a naugthy boy is quite strong :p

File: tests/Jenkins/dirac_ci.sh (hunk 4 above)
Comment (chaen) [reply tests/Jenkins/dirac_ci.sh]: Yes, but what happen when DIRACOS is used ?

Comment (fstagni) [reply tests/Jenkins/dirac_ci.sh]: IIUC with DIRACOS we don't need the "-t server" switch, at least not in Jenkins where the directory structure used is simple (no "pro" nor "old" links).

Comment (chaen) [reply tests/Jenkins/dirac_ci.sh]: TBH I don't exactly know the impact of the `server`, `fullserver` and `client` option is, besides External package

**Comment by fstagni** [comment]
Review, please.

**Pull request merged**